### PR TITLE
Smallpatches

### PR DESCRIPTION
**Pull request by InfusOnWoW**

### CODE REVIEW

**Comment by InfusOnWoW** [comment]
I think if we add fonts, they should font variants, they should all come from the same family. I've choosen the Fira Sans Medium +Black and Fire Condensed Medium + Black variants. The blacks ones are extra-bold variants, which does work nicely for our user cases.

File: WeakAuras/BuffTrigger2.lua [hunk 1]
```diff
@@ -260,17 +260,134 @@ local function MatchesTriggerInfoMulti(triggerInfo, sourceGUID)
...
+        matchDataChanged[id][triggernum] = true
+      end
+    end
+  end
+end
+
+---@class TooltipHelper
+---@field count number
+---@field tracks table<string, table<fun(data: any), any>>
+local TooltipHelper = {
```
Comment (mrbuds) [inline WeakAuras/BuffTrigger2.lua:284]: This whole block can be preceded by `if newAPI then`

File: WeakAuras/BuffTrigger2.lua [hunk 2]
```diff
@@ -2414,6 +2528,8 @@ function BuffTrigger.UnloadAll()
     PerUnitFrames:UnregisterAll()
   end
 
+  TooltipHelper:Clear()
```
Comment (mrbuds) [inline WeakAuras/BuffTrigger2.lua:2531]: `if newAPI then`

File: WeakAuras/BuffTrigger2.lua [hunk 3]
```diff
@@ -260,17 +260,134 @@ local function MatchesTriggerInfoMulti(triggerInfo, sourceGUID)
...
     matchData.tooltip, matchData.tooltip1, matchData.tooltip2, matchData.tooltip3, matchData.tooltip4 = tooltip, tooltip1, tooltip2, tooltip3, tooltip4
+
+    local oldDataInstanceId = matchData.dataInstanceID
+    matchData.dataInstanceID = dataInstanceID
+    if dataInstanceID then
+      TooltipHelper:Track(dataInstanceID, matchData)
+    end
+    if oldDataInstanceId then
+      TooltipHelper:Untrack(oldDataInstanceId, matchData)
+    end
```
[the remark this answers is not in the document]
Comment (InfusOnWoW) [reply WeakAuras/BuffTrigger2.lua:390]: That doesn't seem to happen, every call to C_TooltipInfo.GetUnitBuffByAuraInstanceID returns a new dataInstanceId without any reuse.

**Review by mrbuds — APPROVED** [review]

**Pull request merged**